src/scripts/components/VideoListItem.js

```diff
@@ -149,7 +149,7 @@ class VideoListItem extends Component<Props, void> {
         statusMessage = 'Your video could not be uploaded'
       } else if (video.transcodingStatus.name === 'failed') {
         statusMessage = 'Your video could not be transcoded'
-      } else if (video.transcodingStatus.name === 'running') {
+      } else if (video.uploadStatus.name === 'success') {
         statusMessage = 'Transcoding your video'
```

Comment:
Can you please move this status message generation to its own instance method in the component? Ideally, mostly to keep things clean and approachable, we do not want any complex/multilayered logic in `render`.